app/Models/User.php

```diff
@@ -21,6 +21,8 @@ class User extends Authenticatable
         'name',
         'email',
         'password',
+        'phone',
```

Comment:
Please do not add new functionality, this pull request means to be about the upgrade.

Reply:
oh yes, just to become comfortable in adding new thing, to modified for as needed. I've added on all part required for it.